packages/ng-schematics/src/prompt/SchematicsPromptSession.ts

```diff
@@ -13,6 +13,15 @@ export class SchematicsPromptSession extends BasePromptSession {
 	public context: SchematicContext;
 	public projectName: string;
 
+	public get userAnswers(): Map<string, any> {
+		return this._userAnswers;
+	}
+	public set userAnswers(v: Map<string, any>) {
+		this._userAnswers = v;
+	}
+
+	private _userAnswers: Map<string, any>;
```

Comment:
Does this really need to be a private property w/ accessors?

Reply:
I asked as well 😁 @wnvko Mentioned something about the test spy on property messing up without accessors. Will try without it just to check

Reply:
done